subsys/bluetooth/audio/shell/audio.h

```diff
@@ -59,6 +63,20 @@ struct unicast_stream {
 	struct bt_codec_qos qos;
 };
 
+struct broadcast_stream {
+	struct bt_cap_stream stream;
+	struct bt_codec_data data;
+};
+
+struct broadcast_source {
+	union {
+		struct bt_bap_broadcast_source *bap_source;
+		struct bt_cap_broadcast_source *cap_source;
+	};
```

Comment:
I do not see `cap_source` referenced in this PR.


```suggestion
	struct bt_bap_broadcast_source *bap_source;
```

Reply:
> I do not see cap_source referenced in this PR.

It will be used as part of https://github.com/zephyrproject-rtos/zephyr/issues/55511

I can wait with adding it until then if you want